### ISSUE TITLE
Scoreboard

### ISSUE THREAD

**Issue by SirenDrowned** [StoneworksMC/War#3]
- **Please describe what you want in this suggestion.**
 
A live scoreboard where people can see the current score/kills etc, currently in wars you have to wait for the chat update that happens like once every 10 minutes. It's a lot easier to strategize when you can see the current score easily
- **How should this feature interact with players?**
Maybe this could be done by utilizing boss bars at the top of the screen, or could just be a scoreboard on the side.

**Comment by Matt-MX** [comment]
Yup we plan to use scoreboards while in a war zone, it will be of course toggleable.
Capping a zone using boss bars to indicate progress etc

**Comment by darkcoder07** [comment]
I like the QoL of this idea, but waiting for the chat update did make for some really fun suspense...

Maybe some specific things like boss bars for captures could be real-time or close to it while a scoreboard could update every 5, 3, or 4 minutes or so? Fast but not instantly fast.

**Comment by Matt-MX** [comment]
> I like the QoL of this idea, but waiting for the chat update did make for some really fun suspense...
> 
> Maybe some specific things like boss bars for captures could be real-time or close to it while a scoreboard could update every 5, 3, or 4 minutes or so? Fast but not instantly fast.

It could be argued that it's misleading to do this, I think players would prefer real-time updates tbh

**Comment by darkcoder07** [comment]
Possibly, if it wasn't common knowledge. Instant information access feels wrong to me in this context*, but that's ultimately opinion-based so real-time updates could work too. 

*As in, in an immersive medieval-style battlefield that SW wars were, at least, trying to be. In any case hopefully the scoreboard can be light, since a large one would be annoying for some people (I personally have the regular SW one toggled off).

**Comment by Matt-MX** [comment]
> hopefully the scoreboard can be light

Yeah I don't like big ones, I'll be keeping it concise and only showing info that they need
